src/axom/quest/SamplingShaper.hpp

```diff
@@ -144,26 +180,57 @@ class InOutSampler
       pos_coef->GetValues(i, m);
       inout->GetValues(i, res);
 
-      for(int p = 0; p < nq; ++p)
+      if(projector)
       {
-        const SpacePt pt(m.GetColumn(p), dim);
-        const bool in = m_octree->within(pt);
-        res(p) = in ? 1. : 0.;
+        for(int p = 0; p < nq; ++p)
+        {
+          const ToPoint pt = projector(FromPoint(m.GetColumn(p), dim));
+          const bool in = m_octree->within(pt);
+          res(p) = in ? 1. : 0.;
+
+          // SLIC_INFO(axom::fmt::format("[{},{}] Pt: {}, In: {}", i,p,pt, (in? "yes" : "no") ));
```

Comment:
Why are the SLIC_INFO calls commented out? Should they be removed?

Reply:
They're debugging comments -- I'll remove them